src/templ.ml

```diff
@@ -789,6 +789,8 @@ and eval_simple_variable conf =
         (fun c (k, v) -> c ^ k ^ "=" ^ v ^ ";")
         c l
   | "version" -> Version.txt
+  | "commit" -> Commit.nbr
+  | "commit_date" -> Commit.date
```

Comment:
I wonder if this difference between commit date and compilation date is necessary ?